lib/conjur/api/resources.rb

```diff
@@ -166,5 +166,19 @@ def resources opts = {}
     def global_privilege_permitted? privilege
       resource(GLOBAL_PRIVILEGE_RESOURCE).permitted? privilege
     end
+
+    def resources_permitted? kind, identifiers, privilege
+      options = {
+        privilege: privilege,
+        identifiers: identifiers
+      }
+      resp = RestClient::Resource.new(Conjur::Authz::API.host, credentials)["#{Conjur.account}/resources/#{kind}?check=true"].post(options)
```

Comment:
We want an exception to be raised, right? I believe RestClient does so (though I'll double-check).